PsfRuntime/readme.md

```diff
@@ -57,3 +57,9 @@ The PSF Runtime then calls `PSFInitialize` within a Detours transaction, failing
 
 ## Runtime Requirements
 As a part of its initialization, the PSF Runtime queries information about its environment that it then caches for later use. A few examples include parsing the `config.json`, caching the path to the package root, and caching the package name, among a couple other things. If any of these steps fail, e.g. because something is not present/cannot be found or any other failure, then the PSF Runtime dll will fail to load, which likely means that the process fails to start. Note that this implies the requirement that the application be running with package identity. There have been past conversations on adding support for a "debug" mode that works around this restriction (e.g. by using a fake package name, executable directory as the package root, etc.), but its benefit is questionable and has not yet been implemented.
+
+## Argument Redirection

```

Comment:
This content looks more appropriate for Release notes and not for Readme. This redirection is going to happen internally without user having any control over it. In case if we still decide to have it here, we would need to reword it.